Algorithms.Tests/Strings/MinCostStringConversionTests.fs

```diff
@@ -1,22 +1,23 @@
-﻿namespace Algorithms.Tests.Strings
-
-open Microsoft.VisualStudio.TestTools.UnitTesting
-open Algorithms.Strings
-
-[<TestClass>]
-type MinCostStringConversionTests () =
-
-    [<TestMethod>]
-    [<DataRow("abbbaba", "abbba")>]
-    [<DataRow("ababa", "ababa")>]
-    member this.assembleTransformation (ops:string list, i:int, j:int, expected:string list) =
-        let actual = MinCostStringConversion.assembleTransformation(ops, i, j)
-        Assert.AreEqual(expected, actual)
-
-    [<TestMethod>]
-    [<DataRow("abbbaba", "abbba")>]
-    [<DataRow("ababa", "ababa")>]
-    member this.assembleTransformation (sourceString:string, destinationString:string, copyCost:int, replaceCost:int, deleteCost:int, insertCost:int, expected:int list * string list) =
-        let actual = MinCostStringConversion.computeTransformTables(sourceString,destinationString,copyCost,replaceCost,deleteCost,insertCost)
-        Assert.AreEqual(expected, actual)
-
+﻿namespace Algorithms.Tests.Strings
+
+open Microsoft.VisualStudio.TestTools.UnitTesting
+open Algorithms.Strings
+
+// FIXME
+// [<TestClass>]
+// type MinCostStringConversionTests () =

```

Comment:
This is failing the tests, as explained on https://github.com/TheAlgorithms/F-Sharp/issues/25.

Reply:
Can you fix this test?

Reply:
I think I can. Found it written in Python too -> https://github.com/TheAlgorithms/Python/blob/master/strings/min_cost_string_conversion.py

Reply:
I'm not really sure what to use for the test inputs.

Reply:
This Python example is raising an exception too `System.ArgumentException: The index was outside the range of elements in the list. (Parameter 'n')`.

Reply:
Alright. This PR is for running tests for CI. It does not require correction of the test. I'll merge your PR
Please open an issue for this test issue.

Reply:
Done -> https://github.com/TheAlgorithms/F-Sharp/issues/27